pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -165,9 +165,14 @@ def fractional_octave_bands(
         num_fractions,
         sampling_rate=None,
         frequency_range=(20.0, 20e3),
-        order=14):
+        order=14,
+        check_tolerance_class=1):
```

Comment:
I'm not sure if I'm a great fan of the additional parameter here.
Maybe we can get by with giving the user a warning if the filters are not class 1/2 compliant instead of having to define a class and then getting an error if the filters do not comply.